components/layout/display_list_builder.rs

```diff
@@ -2124,7 +2124,7 @@ impl FragmentDisplayListBuilding for Fragment {
         // to back).
 
         // Shadows
-        for shadow in text_shadows.iter().rev() {
+        for shadow in text_shadows.iter() {
```

Comment:
I'm assuming this is just because of a WR API change. If the reasoning is deeper, please explain.